### DISCUSSION TITLE
Indsætte frivilliges navn i beskeder

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#11] [Bugs] [answered]
Vi møder et ønske om at kunne indsætte et dynamisk felst med den frivilliges navn i en besked, der udsendes til mange. F.eks. ved at man kan få den frivilliges navn indsat ved at skrive [UserName] eller lignende. 

Er det muligt? Og er det en god ide?

**Comment by antonlund18** [comment, marked as answer]
Det kan godt lades at gøre, og der er som sådan ikke noget galt i det. Men det er nok en lidt større opgave. 

Jeg har oprettet et issue på det: #468
